build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -163,27 +163,26 @@ def convert_directory(directory_path, write_files, allow_partial_conversion,
   if not os.path.isfile(build_file_path):
     return Status.NO_BUILD_FILE
 
-  copyright_line = f"# Copyright {datetime.date.today().year} Google LLC"
-  write_allowed = write_files
   if os.path.isfile(cmakelists_file_path):
     with open(cmakelists_file_path) as f:
       for i, line in enumerate(f):
-        if line.startswith("# Copyright"):
-          copyright_line = line.rstrip()
         if EDIT_BLOCKING_PATTERN.search(line):
           if verbosity >= 1:
             log(f"Skipped. line {i + 1}: '{line.strip()}' prevents edits.",
                 indent=2)
           return Status.SKIPPED
 
+  header = (f"# Autogenerated from {rel_build_file_path} by\n"
+            f"# {repo_relpath(os.path.abspath(__file__))}")
```

Comment:
nit: these sort of "autogenerated by x" comments are typically followed by at least a newline, and sometimes are also wrapped in something more attention-grabbing like `#######`

Current:
```
# Autogenerated from iree/hal/BUILD by
# build_tools/bazel_to_cmake/bazel_to_cmake.py
iree_add_all_subdirs()
```

Suggestion 1:
```
# Autogenerated from iree/hal/BUILD by
# build_tools/bazel_to_cmake/bazel_to_cmake.py

iree_add_all_subdirs()
```

Suggestion 2:
```  
###############################################################################
# Autogenerated from iree/hal/BUILD by
# build_tools/bazel_to_cmake/bazel_to_cmake.py
###############################################################################

iree_add_all_subdirs()
```

Reply:
Woo Scott's back. I incorporated this suggestion into https://github.com/google/iree/pull/4943